modules/templates/helper.go

```diff
@@ -1003,6 +1004,33 @@ func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteNa
 	return a
 }
 
+func pushMirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string) remoteAddress {
```

Comment:
I think it's better to avoid creating a new function for this and instead add a control argument to `mirrorRemoteAddress` that will ignore `m.OriginalURL` if the control argument is set.

Reply:
OK. At the beginning, I also thought like this. I saw that `CONTRIBUTING.md` point out `make small pull requests` and `don't make changes unrelated to your PR`, so I added a method. Later, I will add a control argument to judge whether to ignore `m.OriginalURL`.

Reply:
The adjustment has been completed.